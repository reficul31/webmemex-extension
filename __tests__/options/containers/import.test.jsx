import React from 'react'
import ImportContainer from 'src/options/containers/import'
import renderer from 'react-test-renderer'

describe('ImportContainer snapshot tests', () => {
    test('should test against the given snapshot', () => {
        const component = renderer.create(
            <ImportContainer />
        )
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})
