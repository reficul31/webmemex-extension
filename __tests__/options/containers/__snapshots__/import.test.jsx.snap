// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImportContainer snapshot tests should test against the given snapshot 1`] = `
<div>
  <h1
    className="routeTitle"
  >
    Import
  </h1>
  <p>
    Import specific components to come in here.
  </p>
</div>
`;
